scripts/install_prerequisites.sh

```diff
@@ -42,7 +42,7 @@ pip install --upgrade pip==21.2.3
 sudo apt-get install make
 sudo apt-get install build-essential
 sudo apt-get install libssl-dev
-sudo apt-get install lib1g-dev
+sudo apt-get install zlib1g-dev
```

Comment:
I didn't even know this file existed when I was following the installation instructions for Oppia. Would it be possible to add it to [this page](https://github.com/oppia/oppia/wiki/Installing-Oppia-%28Linux%3B-Python-3%29#setup-a-virtual-environment)?

Reply:
It is already mentioned there ![image](https://user-images.githubusercontent.com/10142938/197190465-f966c534-5845-4ca5-9691-8e9c96d68761.png)